online-inference/tensorizer-isvc/benchmark/load_test_async.py

```diff
@@ -0,0 +1,87 @@
+import time
+import asyncio
+import random
+import requests
+import aiohttp
+from argparse import ArgumentParser
+
+from asgiref import sync
+
+def timed(func):
+    """
+    records approximate durations of function calls
+    """
+    def wrapper(*args, **kwargs):
+        start = time.time()
+        print('{name:<30} started'.format(name=func.__name__))
+        result = func(*args, **kwargs)
+        duration = "{name:<30} finished in {elapsed:.2f} seconds".format(
+            name=func.__name__, elapsed=time.time() - start
+        )
+        print(duration)
+        timed.durations.append(duration)
+        return result
+    return wrapper
+
+timed.durations = []
+
+@timed
+def async_requests_get_all(urls):
+    """
+    asynchronous wrapper around synchronous requests
+    """
+    session = requests.Session()
+    # wrap requests.get into an async function
+    def get(url):
+        return session.get(url).status_code
+    async_get = sync.sync_to_async(get)
+
+    async def get_all(urls):
+        return await asyncio.gather(*[
+            async_get(url) for url in urls
+        ])
+    # call get_all as a sync function to be used in a sync context
+    return sync.async_to_sync(get_all)(urls)
```

Comment:
This seems to be dead code superseded by `async_aiohttp_get_all`. Should this still be here?